src/js/controllers/walletHome.js

```diff
@@ -1225,45 +1225,59 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
     return actions.hasOwnProperty('create');
   };
 
-  this._doSendAll = function(amount) {
+  this._doSendMax = function(amount) {
     this.setForm(null, amount, null);
   };
 
-  this.sendAll = function(totalBytesToSendMax, availableBalanceSat) {
+  this.sendMax = function() {
     var self = this;
-    var availableMaxBalance;
-    var feeToSendMaxStr;
+    var fc = profileService.focusedClient;
     this.error = null;
     this.setOngoingProcess(gettextCatalog.getString('Calculating fee'));
 
     feeService.getCurrentFeeValue(function(err, feePerKb) {
       self.setOngoingProcess();
+
       if (err || lodash.isNull(feePerKb)) {
         self.error = gettext('Could not get fee value');
         return;
       }
 
-      var feeToSendMaxSat = parseInt(((totalBytesToSendMax * feePerKb) / 1000.).toFixed(0));
-      if (availableBalanceSat > feeToSendMaxSat) {
-        self.lockedCurrentFeePerKb = feePerKb;
-        availableMaxBalance = strip((availableBalanceSat - feeToSendMaxSat) * self.satToUnit);
-        feeToSendMaxStr = profileService.formatAmount(feeToSendMaxSat) + ' ' + self.unitName;
-      } else {
-        self.error = gettext('Not enought funds for fee');
-        return;
-      }
+      var opts = {};
+      opts.feePerKb = feePerKb;
 
-      var msg = gettextCatalog.getString("{{fee}} will be deducted for bitcoin networking fees", {
-        fee: feeToSendMaxStr
-      });
+      var config = configService.getSync();
+      if (!config.wallet.spendUnconfirmed)
+        opts.excludeUnconfirmedUtxos = 1;
```

Comment:
`opts.excludeUnconfirmedUtxos` should be a boolean variable. So doing `opts.excludeUnconfirmedUtxos = !config.wallet.spendUnconfirmed` should be ok.